src/styles/blocks/header.scss

```diff
@@ -0,0 +1,43 @@
+.header {
+  // display: none;
+  background-image: url(../images/header-bg.png);
+  background-size: cover;
+  background-position: center;
+
+  @include on-desktop {
+    background-image: url(../images/header-bg-2x.png);
+  }
+
+  &__content {
+    height: 100vh;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+  }
+
+  &__bottom {
+    padding-bottom: 40px;
+
+    @include on-tablet {
+      padding-bottom: 64px;
+    }
+  }
+
+  &__title {
+    font-size: 36px;
+    font-weight: 700;
+    line-height: 36px;
```

Comment:
if the value of `font-size` equals value of `line-height`
you can set `line-height: 1`